pkg/operator/workload_controller_openshiftapiserver_v311_00.go

```diff
@@ -287,8 +297,8 @@ func manageOpenShiftAPIServerDaemonSet_v311_00_to_latest(client appsclientv1.Dae
 	return resourceapply.ApplyDaemonSet(client, recorder, required, resourcemerge.ExpectedDaemonSetGeneration(required, generationStatus), forceRollingUpdate)
 }
 
-func manageAPIServices_v311_00_to_latest(client apiregistrationv1client.APIServicesGetter) ([]*apiregistrationv1.APIService, error) {
-	apiServiceGroupVersions := []schema.GroupVersion{
+func latestManagedAPIServiceGroupVersions() []schema.GroupVersion {
+	return []schema.GroupVersion{
```

Comment:
this can be a var, why make it a function?

Reply:
converted to a var